extension/src/time_vector/pipeline/map.rs

```diff
@@ -566,6 +566,105 @@ mod tests {
         });
     }
 
+    #[pg_test]
+    #[should_panic = "division by zero"]
```

Comment:
Interesting.  For some reason, we thought we had to use `#[pg_test(error = "...")]` and we could not get that to work under github actions even though it works locally on mac and linux.  I'll have a look at resurrecting all those disabled tests...

Reply:
Support for more test attributes is [new in 0.5.0](https://github.com/tcdi/pgx/pull/613). I'm surprised it ever worked prior to 0.5.0.